### PR TITLE
fix typo to fix uninstalled state

### PR DESCRIPTION
**Pull request by yeoldegrove**

### CODE REVIEW

**Comment by yeoldegrove** [comment]
@arbulu89 please review and merge

File: salt-shaptools.changes [hunk 1]
```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Wed Jul 21 07:55:12 UTC 2021 - Eike Waldt <waldt@b1-systems.de>
+
+- fix typo to fix uninstalled state
```
Comment (arbulu89) [inline salt-shaptools.changes:4]: If you are going to increase the version number, specify it in the changes here please

**Review by arbulu89 — APPROVED** [review]
Thanks @yeoldegrove ,
It's a pity that we don't have the correct asserts in the tests... 
This should be easy to catch

File: salt-shaptools.changes (hunk 1 above)
Comment (yeoldegrove) [reply salt-shaptools.changes]: @arbulu89 done

**Comment by arbulu89** [comment]
@yeoldegrove , Could you check why the CI tests are failling?

**Pull request merged**